app/presenters/menu/default_menu.rb

```diff
@@ -109,6 +109,25 @@ def container_menu_section
         ])
       end
 
+      def middleware_menu_section
+        Menu::Section.new(:mdl, N_("Middleware"), [
```

Comment:
Please make all the `Menu::Item.new` calls consistent in style.  One is all one line; another has a parameter each on a separate line; the last one mixes styles.  Consistency is important!